projects/angle/addresses.js

```diff
@@ -109,7 +109,142 @@ const guardianAddress = {
+    },
+    polygon: {
+        EURA: '0xE0B52e49357Fd4DAf2c15e02058DCE6BC0057db4',
+        USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+    },
+    avax: {
+        sBUSD: '0xf04d3a8eb17b832fbebf43610e94bdc4fd5cf2dd',
+        EURA: '0xAEC8318a9a59bAEb39861d10ff6C7f7bf1F96C57',
+        USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+    },
+    xdai: {
+        EURA: '0x4b1E2c2762667331Bc91648052F646d1b0d35984',
+        stEUR: '0x004626A008B1aCdC4c74ab51644093b155e59A23',
+        USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+        stUSD: '0x0022228a2cc5E7eF0274A7Baa600d44da5aB5776',
+    },
+    bsc: {
+        TUSD: '0x40af3827f39d0eacbf4a168f8d4ee67c121d11c9',
+        EURA: '0x12f31B73D812C6Bb0d735a218c086d44D5fe5f89',
+        USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+    },
+    celo: {
+        EURA: '0xC16B81Af351BA9e64C1a069E3Ab18c244A1E3049',
+        stEUR: '0x004626A008B1aCdC4c74ab51644093b155e59A23',
+        USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+        stUSD: '0x0022228a2cc5E7eF0274A7Baa600d44da5aB5776',
+    },
+    polygon_zkevm: {
+        EURA: '0xA61BeB4A3d02decb01039e378237032B351125B4',
+        USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+    },
+    mantle: {
+        EURA: '0xA61BeB4A3d02decb01039e378237032B351125B4',
+        USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+    },
+    linea: {
+        EURA: '0x1a7e4e63778B4f12a199C062f3eFdD288afCBce8',
+        USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+        base: {
+            EURA: '0xA61BeB4A3d02decb01039e378237032B351125B4',
+            USDA: '0x0000206329b97DB379d5E1Bf586BbDB969C63274',
+        },
+    },
+}
+
+const treasuryTokens = (tokenSymbols) => {
+    const treasury = {}
+    tokenSymbols.forEach((tokenSymbol) =>
+        chains.forEach((chain) => {
+            const token =
+                ADDRESSES?.[chain]?.[tokenSymbol] ??
+                tokens?.[chain]?.[tokenSymbol]
+
+            if (!treasury?.[chain]) treasury[chain] = []
+            if (token) treasury?.[chain].push(token)
+        })
+    )
+    return treasury
+}
+const stablecoin = {
+    EUR: {
+        transmuter: '0x00253582b2a3FE112feEC532221d9708c64cEFAb',
+        treasury: '0x5f9F41497f9e11fd7D4c4B067413199682eE2CFF',
+        treasuryTokens: treasuryTokens([
+            'USDC',
+            'USDT',
+            'CRV',
+            'CVX',
+            'DAI',
+            'EURA',
+            'WETH',
+            'bC3M',
+            'EURC',
+            'VELO',
+            'bERNX',
+            'steakUSDC',
+            'bIB01',
+        ]),
+        genesis: 1691656362,
+    },
+    USD: {
+        transmuter: '0x222222fD79264BBE280b4986F6FEfBC3524d0137',
+        treasury: '0x57eedCB68445355e9C11A90F39012e8d4AAA89Fc',
+        treasuryTokens: treasuryTokens([
+            'USDC',
+            'USDT',
+            'CRV',
+            'CVX',
+            'DAI',
+            'USDA',
```

Comment:
@clement4saunier thanks for putting up with my childish request, the PR is mostly good, I will merge it and then exclude USDA & EURA from tvl since backing of it is already included in tvl